packages/blockchain-wallet-v4-frontend/src/data/components/interest/selectors.ts

```diff
@@ -14,6 +17,32 @@ export const getInterestEligible = (state: RootState) => state.components.intere
 
 export const getInterestInstruments = (state: RootState) => state.components.interest.instruments
 
+// If the user does not have a rewards balance, move the preferredCoins list to the top of the list (even better if this is done by backend)
+// If the user has a rewards balance, move those instruments to the top of the list and merge with the preferredCoins list
+export const getInstrumentsSortedByBalance = createSelector(
+  getInterestInstruments,
+  getInterestAccountBalance,
+  (insturments, balances) => {
+    const remoteBalances = balances.getOrElse({})
+    const remoteInstruments = insturments.getOrElse([]) as InterestInstrumentsType
```

Comment:
i'd rename these since they arent remotes after calling a selector and/or the `getOrElse`.